### PR TITLE
Feat/status codes

### PR DESCRIPTION
**Pull request by atulbhatt-system32**
### Upcoming Features
- [x] Persist API response to prevent API calls on all pages
- [ ] Find Code - Multiple Codes Selection
- [x] Quiz Page

https://github.com/hiteshchoudhary/apihub/assets/38177419/49162ff9-a565-4dbe-b4d8-4605d1205769

https://github.com/hiteshchoudhary/apihub/assets/38177419/5419efc2-2b13-409c-b043-55b7c29eaddf

### CODE REVIEW

**Comment by wajeshubham** [comment]
@atulbhatt-system32 any update on this? Kindly let us know.

**Comment by atulbhatt-system32** [comment]
@wajeshubham ready for Review

File: package-lock.json
Comment (wajeshubham) [inline package-lock.json:1]: We are using `yarn`. Please read `README.md` and `CONTRIBUTING.md` file carefully before starting the contribution. Please use yarn for package installation. Also, this is a frontend contribution this must not affect core freeapi code files. So, addition of `package-lock.json` is irrelevant here.

File: package.json
Comment (wajeshubham) [inline package.json:1]: This file must not be updated as this is a frontend contribution and this package.json belongs to the core freeapi project.,

**Review by wajeshubham — CHANGES_REQUESTED** [review]
Kindly go through the review comments and it would be great if we have some demo video of the application.

Thanks

File: package-lock.json
Comment (atulbhatt-system32) [reply package-lock.json:1]: Hey @wajeshubham thanks for the feedback. I actually read the contributing.md but my bad I did this blunders even then. Sure, I will go through the contributing.md once more and make necessary updates.

**Comment by atulbhatt-system32** [comment]
> Kindly go through the review comments and it would be great if we have some demo video of the application.
> 
> Thanks

Definitely will include a video too.

**Comment by wajeshubham** [comment]
@atulbhatt-system32 Is this PR ready for review? I can still see the `package-lock.json` file in the commit. Also, when can we expect quiz part for this app as we would like to include all these features in a single PR. Let us know the timeline.

Thank you

**Comment by atulbhatt-system32** [comment]
@wajeshubham by Next weekend I will add the quiz part too. Tomorrow I will plan on some quiz layout.

**Comment by wajeshubham** [comment]
Ok, if that's the case then we can wait till you finish the quiz app and after the changes are done, I'll review and merge the code. Thanks!

**Comment by wajeshubham** [comment]
@atulbhatt-system32 Any update on this?

**Comment by atulbhatt-system32** [comment]
> @atulbhatt-system32 Any update on this?

@wajeshubham updated just now. Also attached a video for the quiz page.

**Review by wajeshubham — APPROVED** [review]

**Pull request merged**